contrib/utilities/relaunch_simulations.sh

```diff
@@ -0,0 +1,312 @@
+      exit 1
+      ;;
+  esac
+done
+
+# Assign default values if not set
+search_file="${search_file:-./simulations_list.txt}"
+job_file="${job_file:-launch.sh}"
+copy_search_file="${copy_search_file:-"false"}"
+launch_restart="${launch_restart:-"false"}"
+rerun_failed="${rerun_failed:-"false"}"
+set_restart_true="${set_restart_true:-"false"}"
+
+# Check if the search file exists
+if [ ! -f "$search_file" ]; then
+  echo "File not found: $search_file"
+  exit 1
+fi
+
+# Print values to user
+echo "************************************************************"
+echo -e " Running $0 with parameters: \n"
+echo " Search file:                           $search_file"
+echo " Slurm job file:                        $job_file"
+echo " Copy search file:                      $copy_search_file"
+echo " Launch timeout simulations:            $launch_restart"
+echo " Rerun failed simulations:              $rerun_failed"
+echo " Set restart of failed simulations to:  $set_restart_true"
+echo "************************************************************"
+echo "************************************************************"
+
+# Copy search file if requested otherwise, keep it as a temporary copy.
+# The temporary copy is needed since "relaunched_simulations.txt" is overwritten.
+search_file_prefix=$(basename "$search_file" .txt)
+search_file_copy=$(echo "${search_file_prefix}_old.txt")
+cp "$search_file" "$search_file_copy"
+if [ "$copy_search_file" == "true" ]
+then
+  echo " Search file copied as $search_file_copy"
+  echo "************************************************************"
+fi
+
+################################################################################
+# Function that prints the message when no restart subsection is found
+print_no_restart_message() {
+  local prm_file="$1"
+  echo "No \"subsection restart\" was found in $prm_file"
+  echo "The simulation cannot be restarted."
+  echo "Please add the subsection and relaunch the simulation with checkpointing enabled."
+}
+################################################################################
+# List all folders to be checked
+echo -e " SIMULATIONS TO BE CHECKED: \n"
+for simulation_folder in $(cat $search_file)
+do
+  echo " $simulation_folder"
+done
+echo ""
+################################################################################
+# Logging files
+relaunched_simulations_file="relaunched_simulations.txt"
+found_timeout_file="timeout_simulations.txt"
+found_failed_file="failed_simulations.txt"
+
+# Clear logging files
+rm -f "$found_timeout_file" "$found_failed_file" "$relaunched_simulations_file"
+
+while IFS= read -r simulation_folder
+do
+  simulation_folder=$(echo "$simulation_folder" | tr -d '\r'| sed 's/[[:space:]]*$//')
+  echo "************************************************************"
+  echo " Checking simulation:"
+  echo "  $simulation_folder"
+
+  # Move into folder if it exists
+  if [ -d "$simulation_folder" ]
+  then
+    cd "$simulation_folder" || exit
```

Comment:
the || exit is to abort? Would you want to display an error message in that case?